packages/gestalt/src/WrapperLink.js

```diff
@@ -0,0 +1,163 @@
+// @flow strict
```

Comment:
@AlbertCarreras could we rename this to `InternalLink`

Reply:
DONE